docs/build_integration.md

```diff
@@ -194,6 +194,11 @@ See the [Provided Environment Variables](https://github.com/google/oss-fuzz/blob
 page in OSS-Fuzz's `base-builder` image documentation for more details on
 environment variables that are available to `build.sh`.
 
+If some environment variables needs to be passed from your CI to the build,
```

Comment:
Remind me please, where is the code that makes this true? I'm not sure this is correct

Reply:
Indeed, that was not correct cf https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/config_utils.py#L69

Reply:
used then https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/config_utils.py#L133 and finally here https://github.com/google/oss-fuzz/blob/6969d2928d91e0302f12c6242776117ecf5574c3/infra/cifuzz/build_fuzzers.py#L94